### PR TITLE
📦 feat: Added validation for DataType of initialize param of class

### PR DESCRIPTION
**Pull request by IvanildoBarauna**
Hello Marian, 

The following changes will be added in missing_mga

- Added validation for data type of input param in class
- Added tests for class initialize
- Smaller adjusts in style code in main component 
- Added notebook for functional tests
- Added Coverage dependencie in requirements.txt

### CODE REVIEW

File: missing_mga/missing_methods.py [hunk 1]
```diff
@@ -10,7 +10,10 @@
 @pd.api.extensions.register_dataframe_accessor("missing")
 class MissingMethods:
     def __init__(self, pandas_obj):
-        self._obj = pandas_obj
+        if not isinstance(pandas_obj, pd.DataFrame):
+            raise TypeError("This package only works with pandas DataFrames.")
+        else:
+            self._obj = pandas_obj
```
Comment (Mgobeaalcoba) [inline missing_mga/missing_methods.py:16]: I like this error control! @IvanildoBarauna 🥇

File: missing_mga/missing_methods.py [hunk 2]
```diff
@@ -107,8 +110,8 @@ def missing_value_heatmap(self):
         Creates a heatmap to visualize the distribution of missing values in the dataset.
         """
         plt.figure(figsize=(10, 6))
-        sns.heatmap(self._obj.isnull(), cmap='viridis', cbar=False)
-        plt.title('Missing Values Heatmap')
+        sns.heatmap(self._obj.isnull(), cmap="viridis", cbar=False)
+        plt.title("Missing Values Heatmap")
```
Comment (Mgobeaalcoba) [inline missing_mga/missing_methods.py:114]: Do you like double quotes better? The simple ones for me! Jaja 😶‍🌫️

**Review by Mgobeaalcoba — APPROVED** [review]
I like your contributions! Go ahead with them!

File: missing_mga/missing_methods.py (hunk 2 above)
Comment (IvanildoBarauna) [reply missing_mga/missing_methods.py:114]: jajaja my code formatter is running on save!

**Pull request merged**